[2021-12-29 23:13:54,898] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2021-12-28T09:00:00+00:00 [queued]>
[2021-12-29 23:13:54,907] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2021-12-28T09:00:00+00:00 [queued]>
[2021-12-29 23:13:54,908] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2021-12-29 23:13:54,908] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2021-12-29 23:13:54,908] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2021-12-29 23:13:55,011] {taskinstance.py:901} INFO - Executing <Task(TwitterOperator): twitter_aluraonline> on 2021-12-28T09:00:00+00:00
[2021-12-29 23:13:55,014] {standard_task_runner.py:54} INFO - Started process 37083 to run task
[2021-12-29 23:13:55,050] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'twitter_dag', 'twitter_aluraonline', '2021-12-28T09:00:00+00:00', '--job_id', '16', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/twitter_dag.py', '--cfg_path', '/tmp/tmpc6nzjt4i']
[2021-12-29 23:13:55,051] {standard_task_runner.py:78} INFO - Job 16: Subtask twitter_aluraonline
[2021-12-29 23:13:55,214] {logging_mixin.py:112} INFO - Running <TaskInstance: twitter_dag.twitter_aluraonline 2021-12-28T09:00:00+00:00 [running]> on host mavipc
[2021-12-29 23:13:55,251] {base_hook.py:89} INFO - Using connection to: id: twitter_default. Host: https://api.twitter.com, Port: None, Schema: None, Login: None, Password: None, extra: XXXXXXXX
[2021-12-29 23:13:55,252] {twitter_hook.py:42} INFO - URL:https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-12-28T09:00:00.00Z&end_time=2021-12-29T09:00:00.00Z
[2021-12-29 23:13:55,350] {logging_mixin.py:112} WARNING - /home/mvrm/alura-projetos/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2021-12-29 23:13:55,709] {twitter_hook.py:42} INFO - URL:https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-12-28T09:00:00.00Z&end_time=2021-12-29T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe17gd9e8if23z9l5e13v3amjum4d
[2021-12-29 23:13:55,710] {logging_mixin.py:112} WARNING - /home/mvrm/alura-projetos/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2021-12-29 23:13:56,127] {twitter_hook.py:42} INFO - URL:https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-12-28T09:00:00.00Z&end_time=2021-12-29T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe17gd7au28mcdex9wtvvb87445fh
[2021-12-29 23:13:56,129] {logging_mixin.py:112} WARNING - /home/mvrm/alura-projetos/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2021-12-29 23:13:56,530] {twitter_hook.py:42} INFO - URL:https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-12-28T09:00:00.00Z&end_time=2021-12-29T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe17gd6zpj5vy5h2zag1ru1eoopkt
[2021-12-29 23:13:56,532] {logging_mixin.py:112} WARNING - /home/mvrm/alura-projetos/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2021-12-29 23:13:56,858] {twitter_hook.py:42} INFO - URL:https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-12-28T09:00:00.00Z&end_time=2021-12-29T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe17gb2vfn0u8a7r4s6cfnnjrcfwd
[2021-12-29 23:13:56,859] {logging_mixin.py:112} WARNING - /home/mvrm/alura-projetos/datapipeline/.env/lib/python3.8/site-packages/urllib3/connectionpool.py:981: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  warnings.warn(
[2021-12-29 23:13:57,353] {taskinstance.py:1057} INFO - Marking task as SUCCESS.dag_id=twitter_dag, task_id=twitter_aluraonline, execution_date=20211228T090000, start_date=20211230T021354, end_date=20211230T021357
[2021-12-29 23:13:59,775] {local_task_job.py:102} INFO - Task exited with return code 0
